src/style-sheets/generic/_card-content.scss

```diff
@@ -0,0 +1,3 @@
+.card-content {
```

Comment:
seems like this can live in the project-card.scss. Also we try and make variables for any colors we add